gems/pending/MiqVm/miq_scvmm_vm.rb

```diff
@@ -0,0 +1,47 @@
+require 'MiqVm/MiqVm'
+require 'Scvmm/miq_scvmm_vm_ssa_info'
+
+class MiqScvmmVm < MiqVm
+  def getCfg(_snap = nil)
+    cfg_hash = {}
+    # Collect disk information
+    # Call out to the Hyper-V host for Info about the VM.
+    host              = @ost.miq_hyperv[:host]
+    port              = @ost.miq_hyperv[:port]
+    if @ost.miq_hyperv[:domain].nil?
+      user = @ost.miq_hyperv[:user]
+    else
+      user = @ost.miq_hyperv[:domain] + "\\" + @ost.miq_hyperv[:user]
+    end
+    password          = @ost.miq_hyperv[:password]
+    scvmm_info_handle = MiqScvmmVmSSAInfo.new(host, user, password, port)
```

Comment:
This is contrary to the pattern established by the other provider types. Typically, the connection to the API handle is established outside of this object and passed in to initialize, instead of establishing it in initialize. The connection should be established in `connect_to_ems` and passed in via the `ost`. This decouples the establishment of the connection from the instantiation the `MiqScvmmVm`, making it more suitable for generalized use.